java/operator/org.eclipse.theia.cloud.operator/src/main/java/org/eclipse/theia/cloud/operator/TheiaCloudArguments.java

```diff
@@ -118,6 +118,10 @@ public enum BandwidthLimiter {
 	    "--continueOnException" }, description = "Whether the operator will continue to run in case of unexpected exceptions.", required = false)
     private boolean continueOnException;
 
+    @Option(names = {
+	    "--oAuth2ProxyImage" }, description = "The version to use of the quay.io/oauth2-proxy/oauth2-proxy image.", required = false, defaultValue = "latest")
+    private String oAuth2ProxyImage;
```

Comment:
Minor but I think it makes more sense to rename this to `oAuth2ProxyVersion` like in the helm chart to avoid confusion.

```suggestion
    private String oAuth2ProxyVersion;
```